src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="Apple"
```

Comment:
The href attribute 'Apple' is not a valid URL. It should be a valid URL or path, such as '/apple' or 'https://example.com/apple'.